content/docs/integrations/istio.txt

```diff
@@ -1,14 +1,14 @@
 ---
-title: Istio service mesh
+title: Istio sidecar service mesh
```

Comment:
Sidecar mesh isnt tested yet, and I dont want to document our current ideas/workarounds until we are positive they are thoroughly tested. Leaving this file as a .txt currently.

Regardless, still cleaned up the file in its current iteration, and could use a review on what I have so far.